homeassistant/components/luxor/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "FX Luminaire Luxor low voltage controller",
```

Comment:
This should be the integration name, which is a brand/product that generally doesn't need translating